engine/apps/google/client.py

```diff
@@ -37,31 +49,28 @@ def fetch_out_of_office_events(self) -> typing.List[GoogleCalendarEvent]:
         """
         https://developers.google.com/calendar/api/v3/reference/events/list
         """
-
-        def _format_datetime_arg(dt: datetime.datetime) -> str:
-            """
-            https://stackoverflow.com/a/17159470/3902555
-            """
-            return dt.strftime("%Y-%m-%dT%H:%M:%SZ")
-
-        now = _format_datetime_arg(datetime.datetime.now(datetime.UTC))
-
         logger.info(
             f"GoogleCalendarAPIClient - Getting the upcoming {self.MAX_NUMBER_OF_CALENDAR_EVENTS_TO_FETCH} "
             "out of office events"
         )
 
+        now = datetime.datetime.now(datetime.UTC)
+        time_min = utils.datetime_strftime(now)
+        time_max = utils.datetime_strftime(
+            now + datetime.timedelta(days=constants.DAYS_IN_FUTURE_TO_CONSIDER_OUT_OF_OFFICE_EVENTS)
+        )
+
         events_result = (
             self.service.events()
             .list(
                 calendarId="primary",
-                timeMin=now,
-                # timeMax= TODO: should we only fetch out of office events for next X amount of time?
+                timeMin=time_min,
+                timeMax=time_max,
```

Comment:
only consider out of office calendar events for next 30 days (will be sure to document this in forthcoming documentation PR)